debian/rules

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/make -f
+%:
+	dh $@
+
+override_dh_auto_clean:
+override_dh_auto_test:
+override_dh_auto_build:
+override_dh_auto_install:
+	cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_LIBEXECDIR=lib -DDISABLE_UPDATER=1 -DPACKAGING=1 -DUDEV_RULES_PATH=/lib/udev/rules.d -DSYSTEMD_SERVICE_PATH=/lib/systemd/system
```

Comment:
The whole point of SAFE_INSTALL=0 is so that you don't need to do PACKAGING=1 workarounds.